build/ci.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -ev
+
+## Install toolchain
+if [[ "${TRAVIS_DIST}" == "xenial" ]]; then
+    if [[ "${LLVM_VERSION}" == "8" ]]; then
```

Comment:
Could you condense these into one, and just use the ${LLVM_VERSION} variable in the commands? seems like that shouldn't be too hard.